### ISSUE TITLE
Improve switch button layout in generated forms

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#1449]
- The switch button label is on the opposite side at other labels
- The description/help text is not shown

Considerations:
- The ShareExtensionModal and Sharing Tab of the workshop should not be impacted/changed

**Comment by BLoe** [comment]
I think we should be able to use a standard horizontal layout `FieldTemplate` with `as={SwitchButton}` for cases where we don't want to "reverse" the layout like in share modal.

**Comment by BALEHOK** [comment]
> I think we should be able to use a standard horizontal layout `FieldTemplate` with `as={SwitchButton}` for cases where we don't want to "reverse" the layout like in share modal.

I think we can do `FieldTemplate` with `as={SwitchButtonWidget}` (because `SwitchButton` renders its label, it is equivalent to `FieldTemplate`). I.e. we create `SwitchButtonWidget`, use it in `SwitchButton` which remains in ShareExtensionModal and Sharing Tab, and the same widget can be used with `FieldTemplate`.

ShareExtensionModal
```
          <FieldTemplate
              key={organization.id}
              name={organization.id}
              layout="switch"
              label={organization.name}
              value={checked}
              onChange={() => {
                const next = checked
                  ? values.organizations.filter(
                      (x: string) => x !== organization.id
                    )
                  : uniq([...values.organizations, organization.id]);
                setFieldValue("organizations", next);
              }}
          />
```

FormEditor
```
         <FieldTemplate
            name={`${name}.schema.required`}
            label="Required Field?"
            as={SwitchButtonWidget}
            value={isRequired}
            onChange={onRequiredChange}
         />
```

Another option is to add `horizontalReversed` layout and use it with `FieldTemplate` and `SwitchButtonWidget` at ShareExtensionModal and Sharing Tab. Looks more consistent to me.

ShareExtensionModal
```
          <FieldTemplate
              key={organization.id}
              name={organization.id}
              layout="horizontalReversed"
              as={SwitchButtonWidget}
              label={organization.name}
              value={checked}
              onChange={() => {
                const next = checked
                  ? values.organizations.filter(
                      (x: string) => x !== organization.id
                    )
                  : uniq([...values.organizations, organization.id]);
                setFieldValue("organizations", next);
              }}
          />
```

FormEditor
```
         <FieldTemplate
            name={`${name}.schema.required`}
            label="Required Field?"
            layout="horizontal"
            as={SwitchButtonWidget}
            value={isRequired}
            onChange={onRequiredChange}
         />
```

**Comment by BLoe** [comment]
Yep, I basically started doing option #1 already, I have a branch with some code changes. I am moving the layout/label into `FieldTemplate.RenderedSwitch` and converting `SwitchButton` to `SwitchButtonWidget`. I am going to clean up what I have and push it and you can pick up where I left off and make any changes that you want to. I think we should leave it as `layout="switch"` for right now since this is the only thing using the "reverse" layout. If we find more things where we want to use this revered layout in the future, we can create the more abstract `reverserHorizontal` layout, but I don't think we need it yet.

**Comment by BLoe** [comment]
Fixed via #1525
